ui/l10n/en.js

```diff
@@ -425,6 +425,8 @@ var dictionary = {"ICMP.code":"ICMP Code",
 "label.allocated":"Allocated",
 "label.allocation.state":"Allocation State",
 "label.allow":"Allow",
+"label.annotated.by":"Annotator",
+"label.annotation":"Annotation",
```

Comment:
If possible, should we say 'Last commented by' and 'Comment'. The word 'annotation' may be used to highlight some text, the feature here allows admins to put comments on entities/resources... is it too late to reconsider renaming the APIs and naming as `comments`?